src/components/ChartMana/index.js

```diff
@@ -19,14 +19,17 @@ export default React.memo(function ChartMana(props) {
   const [faction, setFaction] = React.useState('*')
   const [level, setLevel] = React.useState(5)
 
+  const activeFaction = faction.split(',')
+  const mainFaction = [...activeFaction].pop()
+
   const cards = React.useMemo(
     () =>
       CARDS.filter(
-        card => !card.token && (faction === '*' || card.faction === faction)
+        card => !card.token && (faction === '*' || activeFaction.includes(card.faction))
       ).map(card => getResolvedCardData({ ...card, level })),
-    [level, faction]
+    [level, faction, activeFaction]
```

Comment:
This no longer looks necessary.
```suggestion
    [level, activeFaction]
```